doc/Telco-PaaS-definition/TelcoPaaSDefChart.md

```diff
@@ -1,10 +1,10 @@
 # XGVela Telco PaaS Definition
 |#|Functions|Description|Software (project name & version)|Telco enhancement point|Tag|Priority|Committer|
 |---|---|---|---|---|---|---|---|
-|1|Container||||Infrastructure||
-|2|Orchestration||||Infrastructure||
-|3|Storage||||Infrastructure||
-|4|Network||||Infrastructure||
+|1|Container|(1) Runtime: support Open Container Initiative (OCI) runtime spec and Kubernetes Container Runtime Interface (CRI) to integrate with K8s for container management; (2) Resource management: memory, CPU, NUMA, HW accelerators; (3) VNF/CNF: support managing VNF and CNF in same cluster|(1) ICN: https://wiki.akraino.org/display/AK/ICN+R3++Release (2) Kubevirt: https://github.com/kubevirt/kubevirt||Infrastructure||
```

Comment:
I think the infrastructure part should not be in the scope of XGVela, but it should use [CNTT RA2](https://github.com/cntt-n/CNTT/tree/master/doc/ref_arch/kubernetes) instead. It should be somewhere mentioned that the Infrastructure is not in the scope of XGVela.